style.css

```diff
@@ -1,12 +1,41 @@
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
 
-* {
+/* * {
   margin: 0;
-  padding: 0;
-}
+} */
```

Comment:
- [ Required ] Please consider removing any commented pieces of code from your code base, commented code makes your code base unprofessional and is considered a sign of code that is not clean, please fix this.